src/login/tests/LoginPage.test.jsx

```diff
@@ -176,26 +177,25 @@ describe('LoginPage', () => {
     );
   });
 
-  it('should update the error message on focus out', () => {
-    const errorState = { email: 'Please enter your Email.', password: 'Please enter your Password.' };
+  it('updates the error state for invalid email; format validation', () => {
+    const errorState = { email: '', password: '' };
```

Comment:
we can add the error message here 
```suggestion
    const errorState = { email: 'The email address you\'ve provided isn\'t formatted correctly.', password: '' };
```

and remove it from here https://github.com/edx/frontend-app-authn/blob/a25695e57645ccf40e49df04247af35f392bacb6/src/login/tests/LoginPage.test.jsx#L186 and https://github.com/edx/frontend-app-authn/blob/a25695e57645ccf40e49df04247af35f392bacb6/src/login/tests/LoginPage.test.jsx#L192